about-ro-crate.md

```diff
@@ -0,0 +1,107 @@
+# About RO-Crate
+
+## What is an RO-Crate?
+
+An RO-Crate is a [Research Object](./background.md#research-object-background) (or _RO_) formed of a collection of data (a _crate_) and a special `ro-crate-metadata.json` file which describes the collection.
+
+The collection may contain any kind of research data - papers, data files, software, references to other research, and so on. It may be a folder full of files, an abstract grouping of connected references, or a combination of both.
+
+The `ro-crate-metadata.json` file (also known as the _RO-Crate Metadata Document_) is a plain text file, readable by humans and machines, that includes metadata for each item within the collection - the authors, license, identifier, and so on. Any folder can be turned into an RO-Crate by adding an `ro-crate-metadata.json` file.
+
+Researchers can distribute their work as an RO-Crate to ensure their data travels with its metadata, so that key components are correctly tracked, archived, and attributed. [something about reproducibility]
+
```

Comment:
One of the aspects of RO Crate that we find highly valuable is the fact that you get all of the relevant context information for each file (study, publications, contacts, provenance), so I like that you are pointing out that data travel with their metadata, this is key.